src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -396,6 +402,19 @@ export default {
 		trackSpotlightDisplayedLoans() {
 			this.trackDisplayedLoans('spotlight', 4, this.spotlightLoans);
 		},
+		check5DollarsBannerCookie() {
+			const currentDate = new Date();
+			const dateCookie = this.cookieStore.get('kvfivedollarsbanner');
+			if (!dateCookie) {
+				this.cookieStore.set('kvfivedollarsbanner', currentDate);
```

Comment:
In many situations we pass in `{ path: '/' }` as the third parameter to `.set`. Is that needed here?

Reply:
What is the default cookie expiration here?

Reply:
From docs, `If omitted, the cookie becomes a session cookie.`

Should we add an expiration time?

Reply:
Do we want it to be a session cookie? We check for 3 days elsewhere, so I'm guessing a session cookie would be long gone by then.

Reply:
Right! Agree by 6 months? it would be there for experiment now, but if becomes a feat we need to extend this expiration time

Reply:
6 months is probably fine, but maybe discuss with the team?

Reply:
Sounds good, thanks!